CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,38 @@
+# Code of Conduct
+
+The **pymovements** community is dedicated to providing a welcoming, inclusive, and harassment-free experience for everyone.
+We expect all participants -- contributors, maintainers, users, and collaborators -- to be respectful and considerate in all project interactions.
+
+## Our Standards
```

Comment:
this header could be removed, instead create a header for encouraged and restricted/inappropriate behavior